server/routes/shared_api/observation/series.py

```diff
@@ -42,17 +42,24 @@
 bp = Blueprint("series", __name__, url_prefix='/api/observations/series')
 
 
-@bp.route('', strict_slashes=False)
+@bp.route('', strict_slashes=False, methods=['GET', 'POST'])
 @cache.cached(timeout=TIMEOUT, query_string=True)
 def series():
   """Handler to get preferred time series given multiple stat vars and entities."""
-  entities = list(filter(lambda x: x != "", request.args.getlist('entities')))
-  variables = list(filter(lambda x: x != "", request.args.getlist('variables')))
+  if request.method == 'POST':
+    entities = request.json.get('entities')
+    variables = request.json.get('variables')
+    facet_ids = request.json.get('facetIds')
+  else:
+    entities = list(filter(lambda x: x != "", request.args.getlist('entities')))
```

Comment:
nit: make this long expression a function..

Reply:
done